.github/workflows/serverless-artillery-tests.yaml

```diff
@@ -1,25 +1,32 @@
-name: Serverless Artillery Users/Id Test
-
-#on:  
-#  push:
-#    branches:
-#      - development
-#      - derek-task1-sprint4
+name: Serverless Artillery Tests
 
 on:
   workflow_run:
-    workflows: ["Serverless Artillery Users Test"]
+    workflows: ["Deploy Function App to Azure"]
     branches: 
       - development
       - derek-task1-sprint5
```

Comment:
We probably do not need to keep - derek-task1-sprint5 you used for testing in to development